### PR TITLE
Changes to allow multi-arch automated kuttl tests

### PR DESCRIPTION
**Pull request by yongja79**
- Enabled Z and P to be tested in the automation

### CODE REVIEW

File: scripts/pipeline/ocp-cluster-e2e.sh [hunk 1]
```diff
@@ -10,7 +10,14 @@ setup_env() {
     # Install kubectl and oc
     curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_CLIENT_VERSION}/openshift-client-linux.tar.gz | tar xvz
     sudo mv oc kubectl /usr/local/bin/
-
+    if [[ "$ARCHITECTURE" == "Z" ]]; then
+    {
+      echo "****** Installing kubectl-kuttl..."
+      curl -L -o kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.15.0/kubectl-kuttl_0.15.0_linux_x86_64
```
Comment (leochr) [inline scripts/pipeline/ocp-cluster-e2e.sh:16]: Platform Z is typically s390x, so wondering if it should be `https://github.com/kudobuilder/kuttl/releases/download/v0.15.0/kubectl-kuttl_0.15.0_linux_s390x` ?

File: scripts/pipeline/ocp-cluster-e2e.sh [hunk 2]
```diff
@@ -330,6 +394,10 @@ parse_args() {
       shift
       readonly INSTALL_MODE="${1}"
       ;;
+        --architecture)
```
Comment (leochr) [inline scripts/pipeline/ocp-cluster-e2e.sh:397]: Fix the indentation

File: scripts/pipeline/runTest.sh [hunk 3]
```diff
@@ -81,3 +82,9 @@ else
     curl -X POST -H 'Content-type: application/json' --data '{"text":"access console at: '$console'"}' $(get_env slack_web_hook_url) </dev/null
     curl -X POST -H 'Content-type: application/json' --data '{"text":"credentials: kubeadmin/'$token'"}' $(get_env slack_web_hook_url) </dev/null
 fi
+
+echo "Cleaning up after tests have be completed"
+echo "switching back to scripts/pipeline directory"
+cd ..
+oc logout
+export CLUSTER_URL=""
```
Comment (leochr) [inline scripts/pipeline/runTest.sh:90]: Is it being reset here for the next test run?

**Review by leochr — COMMENTED** [review]
@yongja79 Thank you. Looks good. Added a couple of comments/questions.

File: scripts/pipeline/ocp-cluster-e2e.sh (hunk 1 above)
Comment (yongja79) [reply scripts/pipeline/ocp-cluster-e2e.sh:16]: Its installing onto the machine that kicks of the tests not the s390x cluster so the image is corrent

File: scripts/pipeline/runTest.sh (hunk 3 above)
Comment (yongja79) [reply scripts/pipeline/runTest.sh:90]: Yes its resetting for the next run

File: scripts/pipeline/ocp-cluster-e2e.sh (hunk 2 above)
Comment (yongja79) [reply scripts/pipeline/ocp-cluster-e2e.sh]: Done

**Review by leochr — APPROVED** [review]
@yongja79 Thanks for the update. Merge when you think it's ready and validated in OnePipeline to kick off and run tests for all 3 platforms (I understand there'll be some storage test failures in the interim)

**Comment by yongja79** [comment]
Completed test run with test failures [Run#1053](https://cloud.ibm.com/devops/pipelines/tekton/c3b2797d-c8ad-4dab-9660-2ec8c0643a76/runs/3e0220d7-af78-40f4-aa40-8b34922a5f48/code-ci-finish/)
Current test run [Run#1056](https://cloud.ibm.com/devops/pipelines/tekton/c3b2797d-c8ad-4dab-9660-2ec8c0643a76/runs/709b15d0-02e5-4c4a-b683-14d22201f1d1/deploy-acceptance-tests)

**Pull request merged**